### PR TITLE
feat: update dataset data profiling extractions table

### PR DESCRIPTION
**Pull request by echl**
# Description

* Include a summary of the changes and the related issue. 
* Include relevant motivation and context.

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue [hunk 1]
```diff
@@ -53,10 +51,13 @@
 					<section>{{ dataset?.username || '-' }}</section>
 				</section>
 			</section>
-			<section class="metadata data-row" v-if="!metadata">
+			<section class="metadata data-row">
 				<section>
 					<header>Source Name</header>
-					<section>{{ dataset?.source || '-' }}</section>
+					<section v-if="dataset.url === 'https://github.com/reichlab/covid19-forecast-hub/'">
```
Comment (dvince2) [inline packages/client/hmi-client/src/components/dataset/tera-dataset.vue:57]: Is this hardcoded URL intended?

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue [hunk 2]
```diff
@@ -218,7 +282,7 @@ function highlightSearchTerms(text: string | undefined): string {
 	return text ?? '';
 }
 
-const dataset: Ref<Dataset | null> = ref(null);
+const dataset = ref<Dataset | null>(null);
```
Comment (dvince2) [inline packages/client/hmi-client/src/components/dataset/tera-dataset.vue:285]: Removed type?

**Review by dvince2 — APPROVED** [review]
Just the one hardcoded URL I'd love to know about, but looks good

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue (hunk 1 above)
Comment (echl) [reply packages/client/hmi-client/src/components/dataset/tera-dataset.vue:57]: Yeah, just for now!

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue (hunk 2 above)
Comment (echl) [reply packages/client/hmi-client/src/components/dataset/tera-dataset.vue:285]: Collateral damage from something I was testing, but it's the same thing, different syntax

**Review by dvince2 — APPROVED** [review]

**Review by mwdchang — CHANGES_REQUESTED** [review]
Why is there a `modelparts/Observables.java` class?

There is already a `modelparts/semantics/Observable.java` class??


I would fix up `modelparts/semantics/Observable.java` (it the the wrong package declarations) rather than trying to 2 classes that are doing the same thing.

File: .eslintrc.cjs [hunk 3]
```diff
@@ -21,6 +21,7 @@ module.exports = {
 		'prefer-destructuring': 'off',
 		'vuejs-accessibility/click-events-have-key-events': 'off',
 		'vuejs-accessibility/label-has-for': 'off',
+		'vuejs-accessibility/form-control-has-label': 'off',
```
Comment (mwdchang) [inline .eslintrc.cjs:24]: What is the point of adding the rule here?

**Comment by YohannParis** [comment]
Here is the file https://askemgroup.slack.com/archives/C03STHFV7K5/p1689262679643979 the one about data

**Review by dvince2 — APPROVED** [review]

**Comment by tangopapatime** [comment]
[meta.webm](https://github.com/DARPA-ASKEM/Terarium/assets/98172806/2cd7ef5b-c54b-4559-a950-d7a0b4040da4)

**Review by YohannParis — APPROVED** [review]

**Pull request merged**